app/src/main/res/values-en/strings.xml

```diff
@@ -998,7 +998,9 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_surface_detailed_answer_impossible_title">Describe surface</string>
     <string name="open_url">Open URL</string>
     <string name="quest_dietType_kosher_name_title">Does %s offer kosher products?</string>
+    <string name="quest_dietType_halal_name_title">Does %s offer halal products?</string>
     <string name="quest_dietType_explanation_kosher">Kosher products must be prepared according to special rules, with kashrut certification agencies marking products in compliance.</string>
+    <string name="quest_dietType_explanation_halal">Halal products must be prepared according to special rules, with Halal certification agencies marking products in compliance.</string>
```

Comment:
AFAIK you need to edit `app/src/main/res/values/strings.xml` as the base file, rather than this one.